core/domain/rights_manager.py

```diff
@@ -285,7 +415,18 @@ def create_new_collection_rights(collection_id, committer_id):
 
 
 def get_collection_rights(collection_id, strict=True):
-    """Retrieves the rights for this collection from the datastore."""
+    """Retrieves the rights for this collection from the datastore.
+
+    Args:
+        collection_id: str. ID of collection.
+        strict: bool. Whether to raise an error if ID is not found.
+
+    Returns:
+        ActivityRights. rights object for the collection.
+
+    Raises:
+        EntityNotFound. collection_id specified is not found.
```

Comment:
Type is (slightly) incorrect, and docstring could be slightly clearer. Please see above notes on the corresponding exploration-related function.